lib/userFactory.js

```diff
@@ -69,6 +69,9 @@ class UserFactory extends BaseFactory {
      * @return {UserFactory}
      */
     static getInstance(config) {
+        if (!instance && (!config || !config.scmPlugin)) {
+            throw new Error('No scm plugin provided to UserFactory');
```

Comment:
Yeah, it's just a big diff. Found it [here](https://github.com/screwdriver-cd/models/pull/92/files#diff-c6dfbb25b49dd3e823a2aa3a6f797f96L174)